src/main/default/lwc/voteTracker/voteTracker.html

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+    <template if:true={finalState}>
+      <template if:true={isBig}>
+        <c-section-title title={finalText}></c-section-title>
+      </template>
+      <template if:false={isBig}>
+        <h3 class="slds-text-title_bold slds-m-vertical_xxx-small">
+          {finalText}
+        </h3>
+      </template>
+      <template if:true={showAbandonRequest}>
+        <div class="slds-m-top_x-small">
+          <lightning-button
+            onclick={handleAbandon}
+            variant="brand-outline"
+            label={abandonLabel}
+          ></lightning-button>
+        </div>
+      </template>
```

Comment:
I don't think the Abandon Request should be part of the voteTracker component.

Or at least the Abandon Request should be its own component and other than containing it, it shouldn't interact with a voteTracker component.

Reply:
I will add it as a separate component 👌